device-types/Ubiquiti/U6-IW.yaml

```diff
@@ -1,28 +1,43 @@
 ---
 manufacturer: Ubiquiti
-model: UniFi U6 In-Wall Access Point
-slug: u6-iw
-part_number: u6-iw-us
+model: Access Point U6 In-Wall
+slug: ubiquiti-u6-iw
+part_number: U6-IW
 u_height: 0
-comments: '[UniFi 6 In-Wall Access Point](https://store.ui.com/products/access-point-wifi-6-in-wall)'
+comments: |
+  [Access Point U6 In-Wall](https://store.ui.com/products/access-point-wifi-6-in-wall)
 is_full_depth: false
+airflow: passive
+weight: 460
+weight_unit: g
 interfaces:
-  - name: Port 1
+  - name: eth0
     type: 1000base-t
-    poe_mode: pse
+    poe_mode: pd
     poe_type: type2-ieee802.3at
-  - name: Port 2
+  - name: eth1
+    label: '1'
     type: 1000base-t
-  - name: Port 3
+    poe_mode: pse
+    poe_type: type2-ieee802.3at
+  - name: eth2
+    label: '2'
     type: 1000base-t
-  - name: Port 4
+    poe_mode: pse
+    poe_type: type2-ieee802.3at
+  - name: eth3
+    label: '3'
     type: 1000base-t
-  - name: Port 5
+    poe_mode: pse
+    poe_type: type2-ieee802.3at
+  - name: eth4
+    label: '4'
     type: 1000base-t
-    poe_mode: pd
+    poe_mode: pse
     poe_type: type2-ieee802.3at
-    mgmt_only: true
-  - name: Wlan 0
+  - name: wlan0
     type: ieee802.11n
-  - name: Wlan 1
+  - name: wlan1
     type: ieee802.11ac
```

Comment:
Where are you getting `ac` from? This is an ax access point

Reply:
```suggestion
    type: ieee802.11ax
```